Misc/NEWS.d/next/Tests/2019-03-19-17-39-25.bpo-31904.QxhhRx.rst

```diff
@@ -0,0 +1,2 @@
+VxWorks uses UTF-8 as the system encoding. So the test case test_utf8_mode.py
+need to be updated to use UTF-8 on VxWorks.
```

Comment:
I propose:

Fix test_utf8_mode on VxWorks: Python always use UTF-8 on VxWorks .

Reply:
I have made the requested changes; please review again